packages/design-system/src/components/Badge/Badge.tsx

```diff
@@ -1,26 +1,28 @@
-import { styled, type DefaultTheme } from 'styled-components';
+import { css, styled, type DefaultTheme } from 'styled-components';
 
 import { Flex, FlexComponent, FlexProps } from '../Flex';
 import { Typography } from '../Typography';
 
-const Base = styled<FlexComponent>(Flex)<{ $size: BadgeSize }>`
-  border-radius: ${({ theme, $size }) => ($size === 'S' ? '2px' : theme.borderRadius)};
-  height: ${({ $size, theme }) => theme.sizes.badge[$size]};
-`;
-
-export type BadgeSize = 'S' | 'M';
+type BadgeSizes = 'S' | 'M';
```

Comment:
For types, singular makes more sense to me. If `size: BadgeSize` I know it's a badge size. If I see `size: BadgeSizes` I expect I can pass in an array of sizes or something.

Reply:
You think? I was super on the fence about this, and i don't really mind – I wanted to find a library that did something i could draw upon, if you feel strongly about it im happy to do it 😄

Reply:
Yeah, I think so, although I feel bad that we've had that "deprecated: use BadgeSizes" and now we're undeprecating it (once you remove the comment) instead

Reply:
I added the deprecation here, so dw lol